src/server/qgsmapserviceexception.h

```diff
@@ -34,15 +36,12 @@
  *  * "OperationNotSupported"
  */
 
-class SERVER_EXPORT QgsMapServiceException : public QgsException
+class SERVER_EXPORT QgsMapServiceException : public QgsOgcServiceException
 {
   public:
-    QgsMapServiceException( const QString& code, const QString& message );
-    QString code() const {return mCode;}
-    QString message() const {return mMessage;}
-  private:
-    QString mCode;
-    QString mMessage;
+    QgsMapServiceException( const QString& code, const QString& message )
```

Comment:
Why not just remove this class totally and document the api break in doc/api_break.dox?